### PR TITLE
docs: Update import in readme

### PR DESCRIPTION
**Pull request by mostlyfabulous**
Path has changed?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ Parameters:
 Creating a Client is even simpler:
 
 ```javascript
-import { Client } from 'ifrau/client.js';
+import { Client } from 'ifrau/client/client.js';
```
Comment (dlockhart) [inline README.md:63]: I don't think so... it's using `exports`, which remaps everything:
https://github.com/Brightspace/ifrau/blob/master/package.json#L7

Comment (grant-cleary) [reply README.md:63]: Yeah, I don't think this should be necessary? If you're using a build system that can't handle `exports` then yes, but I don't think this documentation should necessarily change for that.

Comment (grant-cleary) [reply README.md:63]: Something something webpack? :P

Comment (mostlyfabulous) [reply README.md:63]: I think you're right about webpack 😁 

Oh weird, I couldn't import it into [d2l-class-stream](https://github.com/Brightspace/d2l-class-stream):

```
ERROR in ./node_modules/@d2l/d2l-class-stream-ui/components/d2l-cs-ifrau-client-behavior.js
    Module not found: Error: Can't resolve 'ifrau/client.js' in 'C:\D2L\d2l-class-stream\app\node_modules\@d2l\d2l-class-stream-ui\components'
     @ ./node_modules/@d2l/d2l-class-stream-ui/components/d2l-cs-ifrau-client-behavior.js 1:0-41 10:44-50
     @ ./node_modules/@d2l/d2l-class-stream-ui/components/d2l-cs-app-ready-behavior.js
     @ ./node_modules/@d2l/d2l-class-stream-ui/components/d2l-cs-post-later.js
     @ ./src/components/d2l-cs-app-page-post-later.js
     @ ./src/components/d2l-cs-app.js
     @ ./src/index.js

    ERROR in ./src/components/d2l-cs-app.js
    Module not found: Error: Can't resolve 'ifrau/client.js' in 'C:\D2L\d2l-class-stream\app\src\components'
     @ ./src/components/d2l-cs-app.js 13:0-41 103:40-46
     @ ./src/index.js

    ERROR in chunk 4
    a9f8e40c1a0192f91b01.js
    C:\D2L\d2l-class-stream\app\node_modules\babel-loader\lib\index.js??ref--4!C:\D2L\d2l-class-stream\app\node_modules\string-replace-loader\index.js??ref--6!C:\D2L\d2l-class-stream\app\node_modules\@d2l\d2l-class-stream-ui\components\d2l-cs-app-ready-behavior.js 6aa5ae96c0c970b7d48959f5a4a06058
    Unexpected token (11:44)
    |       readonly: true,
    |       value: function () {
    |         window.ifrauclient = options => new !(function webpackMissingModule() { var e = new Error("Cannot find module 'ifrau/client.js'"); e.code = 'MODULE_NOT_FOUND'; throw e; }())(options);
    |
    |         console.log('ifrau-client-bhr');

    ERROR in chunk index [entry]
    448615fb28324efa4caa.js
    C:\D2L\d2l-class-stream\app\node_modules\babel-loader\lib\index.js??ref--4!C:\D2L\d2l-class-stream\app\node_modules\string-replace-loader\index.js??ref--6!C:\D2L\d2l-class-stream\app\src\index.js 13bbbf9e2f1029f2cb5692606d549665
    Unexpected token (104:40)
    |   },
    |   ready: function () {
    |     window.ifrauclient = options => new !(function webpackMissingModule() { var e = new Error("Cannot find module 'ifrau/client.js'"); e.code = 'MODULE_NOT_FOUND'; throw e; }())(options);
    |
    |     var client = window.ifrauclient({
```

**Pull request closed without merging**